NetCasbin/ThrowHelper.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Diagnostics;
+
+namespace NetCasbin
+{
+    internal static class ThrowHelper
+    {
+        internal static void ThrowNameNotFoundException()
```

Comment:
This method is commented out at `DefaultRoleManager.cs`, do you have any concerns?

Reply:
It will test failed in now test cases. because this API is fail-safe now, we should not change to fail-fast without notice of breaking change.